ec2config.yaml

```diff
@@ -111,6 +109,8 @@ dataverse:
   version: 4.15.1
   wholetale:
     enabled: true
+  dataexplorer:
+    enabled: true
```

Comment:
Interesting! I hope this file is working for you!

Reply:
I updated it on my local copy, destroyed all the data, and now running create_sample_data.py, so more info soon, hopefully.

Reply:
It worked! I get both options here and they each open in their respective tool.

http://ec2-3-80-234-144.compute-1.amazonaws.com/file.xhtml?fileId=255&version=1.0

Hooray!